services/movie.py

```diff
@@ -28,7 +33,8 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Consider using `list[int]` for type hinting `genres_ids` and `actors_ids` for consistency and clarity.